app/views/users/friends.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="container">
+
+<%= render partial: "/shared/profile_bar", locals: { user: @user }%>
+<div class="col-sm-10 col-sm-offset-1">
+
+<% @user.friends.each do |friend| %>
+<div id="friend-pic" class="friends-col">
+  <%= profile_pic(friend) %>
+  <span>
+    <%= link_to friend, friend %>
```

Comment:
tried to access this link. couldn't find it. maybe make the pic itself a link_to too?